ci/tasks/build-ova.yml

```diff
@@ -10,6 +10,9 @@ inputs:
 - name: dispatch
   path: src/github.com/vmware/dispatch
 
+- name: dispatch-ui-binaries
+  path: src/github.com/vmware/dispatch-ui-binaries
```

Comment:
this should be optional, now ova-build tests will fail

Reply:
can make the file in `ova-manifest.json` optional?